firmware/user/modes/mode_ddr.c

```diff
@@ -580,7 +638,40 @@ static void ICACHE_FLASH_ATTR ddrHandleArrows(void)
 static void ICACHE_FLASH_ATTR ddrAnimateNotes(void* arg __attribute__((unused)))
 {
     ddr->NoteIdx = (ddr->NoteIdx + 1) % 4;
-    // testUpdateDisplay();
+}
+
+static void ICACHE_FLASH_ATTR ddrMakeScoreString(ddrWinResult_t curWin, char* scoreText, unsigned int size)
+{
+    const char* typeText;
+    switch (curWin.winType)
+    {
+        default:
+        case DDR_PASS:
+            typeText = "C";
+            break;
+        case DDR_MANY_HIT:
+            typeText = "B";
+            break;
+        case DDR_MANY_PERFECT:
+            typeText = "A";
+            break;
+        case DDR_ALL_HIT:
+            typeText = "AA";
+            break;
+        case DDR_ALL_PERFECT:
+            typeText = "AAA";
+            break;                
+    }
+
+    if (ddr->isNewHighScore)
+    {
+        ets_snprintf(scoreText, size, "*%-3s %09d*", typeText, curWin.score);
```

Comment:
The Tom Thumb font is already not the most readable, and the leading zeros don't help in that regard. `*%-3s %-9d*` may be a little easier on the eyes.

Reply:
This definitely is an improvement. I went ahead with `*%-3s %9d*` to keep the number right justified

Reply:
Makes sense, and you even justified your decision